_opensearch/segment-replication/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Segment Replication Feature
+nav_order: 122
+parent: Opensearch
+has_children: true
+redirect_from:
+  - /opensearch/segment-replication/
+---
+
+## Segment Replication
+
+Opensearch 2.3 contains an experimental version of segment replication.
+
+With segment replication, segment files are copied across shards instead of indexing documents on each shard copy. This improves indexing throughput at the expense of increased network utilization.
```

Comment:
Should we just mention the benefit here (improving index throughput and decreasing data refresh lag), and move the caveat below, after we discuss why this feature will be useful?
Specifically, the caveat can be something like:
Segment replication comes with increased network impact on the primary shard, and may lead to a network bottleneck.

Reply:
Marc and I will sync on wording